Update the new rule page to disable rulename toggle

ryanlerch opened the pull request:
Resolves: https://github.com/fedora-infra/fmn/issues/601

abompard commented on frontend/src/components/NewRuleForm.vue:
```diff
@@ -66,9 +66,10 @@ const handleGenerationRulesChanged = (rules: GenerationRule[]) => {
-            button-class=""
-            input-class="form-control-lg"
+            value=""
+            button-class="fs-3"
```
The `fs-3` class here makes the icon a bit lower than the text. Removing the class makes it look properly aligned. How about removing it?

abompard commented on frontend/src/components/EditableName.vue:
```diff
@@ -30,26 +31,38 @@ onUpdated(() => {
 
 <template>
-  <span :class="{ 'd-none': !editing }">
+  <template v-if="props.noToggle === true">
```
I don't think it's worthwhile to regroup both toggle and noToggle modes in the same component, since they are displaying very different things and no code is actually shared between the two modes, is it?
I would just put the `FormKit` element in the `NewRuleForm`, since it's the only place where it's called with `noToggle`.

ryanlerch replied:
you are right -- i was def ovethinking this one a bit :)

have updated it now to just use a text input vias formkit

ryanlerch committed:
Just use text input on New Rule Form

Previously, we were using the editablename pattern and widget on both
the edit rule and new rule page. As we pretty mush always want to set
the name on the new rule, we now use a text input here instead

Resolves: #601

Signed-off-by: Ryan Lerch <rlerch@redhat.com>
